lib/_http_server.js

```diff
@@ -1027,11 +1027,16 @@ function resOnFinish(req, res, socket, state, server) {
       socket.end();
     }
   } else if (state.outgoing.length === 0) {
-    if (server.keepAliveTimeout && typeof socket.setTimeout === 'function') {
+    const keepAliveTimeout = NumberIsFinite(server.keepAliveTimeout) && server.keepAliveTimeout >= 0 ?
+      server.keepAliveTimeout : 0;
```

Comment:
Why 0 instead of `5000` which is the default?

Reply:
Because I don't want to break. It was undefined pre `keepAliveTimeoutBuffer`.